docs/vscode-devcontainer/Install.md

```diff
@@ -0,0 +1,62 @@
+# Plutus Visual Studio Code DevContioner
+
+This is a docker container configured for use with Visual Studio Code
+that includes ghcide to provide features to make it easier to develop
+Plutus code.
+
+
```

Comment:
Should we note that this focuses on *native* binaries first, e.g. that the plutus applications will end up being native (x86_64-linux) ones?